infra/docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
```

Comment:
A docker-compose file is a good idea, it would be better if you replace the `build' section directly with an image. Although the image doesn't exist on the docker hub, you can use a placeholder for it and I'll upload the image later.